core/src/main/scala/kafka/server/metadata/BrokerMetadataListener.scala

```diff
@@ -259,6 +260,10 @@ class BrokerMetadataListener(
     clientQuotaManager.handleQuotaRecord(record)
   }
 
+  def handleProducerIdRecord(record: ProducerIdsRecord): Unit = {
+    // no-op
```

Comment:
Can you add a short description of why this is a no-op? For example, "Brokers get their IDs from the RPC, so we don't need this" or something like that. Otherwise it looks like a TODO (I know that's not the intention)